scripts/csv-to-json/district.js

```diff
@@ -15,6 +16,17 @@ function toNumber(source, converter) {
   return result;
 }
 
+function _cleanupNumber(source) {
+  let result = trim(source)
+  result = source.replace(',', '')
```

Comment:
```suggestion
  result = source.replace(',', '')
```
```suggestion
  result = source.replace(',', '')
```
this means that `let result = trim(source)` is never taken effect

I think should be 
```
  const result = trim(source).replace(',', '')
```

Reply:
@clairesunstudio Nice catch! Fixed it. Thanks!